File: FRONT/src/app/models/venda.ts
import { FormaPagamento } from './forma-pagamento';
import { ItemVenda } from './item-venda';

export interface Venda {
    vendaId?: number;
    criadoEm?: string;
    itens?: ItemVenda[];
    cliente?: string;
    formaPagamento?: FormaPagamento;
    formaPagamentoId?: number;
}

File: API/Models/Venda.cs
using System;
using System.Collections.Generic;

namespace API.Models
{
    public class Venda
    {
        public Venda() => CriadoEm = DateTime.Now;
        public int VendaId { get; set; }
        public string Cliente { get; set; }
        public List<ItemVenda> Itens { get; set; }
        public DateTime CriadoEm { get; set; }
    }
}